lib/sync/mongodbQueue.js

```diff
@@ -0,0 +1,68 @@
+var mongodbQ = require('mongodb-queue');
+
+/**
+ * Make sure the real queue is created. Otherwise throws an error.
+ */
+function ensureQueueCreated(target) {
+  if (!target.queue) {
+    throw new Error('queue ' + target.queueName + ' is not created yet. Please call queue.create first');
+  }
+}
+
+/**
+ * construct a new queue that is backed by mongodb.
+ * @param {String} name the name of the queue.
+ * @param {Object} opts configuration options for the queue
+ * @param {Object} opts.mongodb an instance of the mongodb connection. This is required.
+ * @param {Number} opts.visibility see https://github.com/chilts/mongodb-queue#visibility---message-visibility-window
+ */
+function MongodbQueue(name, opts) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this 'MongodbQueue' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)